src/main/java/ReviewResponse2.java

```diff
@@ -0,0 +1,25 @@
+public class ReviewResponse2 {
+    private int line;
+    private String comment;
+
+    public ReviewResponse2(int line, String comment) {
```

Comment:
The constructor is responsible for setting both line and comment, adding to the violation of SRP. (Severity: medium)